build/v1/types.go

```diff
@@ -938,6 +938,10 @@ type BuildConfigSpec struct {
 	// When a BuildConfig is created, the 5 most recent failed builds are retained unless this value is set.
 	// If removed after the BuildConfig has been created, all failed builds are retained.
 	FailedBuildsHistoryLimit *int32 `json:"failedBuildsHistoryLimit,omitempty" protobuf:"varint,5,opt,name=failedBuildsHistoryLimit"`
+
+	// UseClusterCA denotes whether or not to use the cluster defined certificate authority
```

Comment:
// UseClusterCA denotes whether the builder container will use CAs from the cluster, or the CAs included in the builder image's own /etc/pki/ca-trust directory.  Setting this to true will overwrite the contents of /etc/pki/ca-trust w/ the CA trust values from the cluster.  Default: false

Reply:
updated in current push